attribute/date_attribute.go

```diff
@@ -8,14 +8,14 @@ import (
 	"github.com/getyoti/yoti-go-sdk/v2/yotiprotoattr"
 )
 
-// TimeAttribute is a Yoti attribute which returns a time as its value
-type TimeAttribute struct {
+// DateAttribute is a Yoti attribute which returns a time as its value
```

Comment:
Think we should update this comment too. Something like this:
```suggestion
// DateAttribute is a Yoti attribute which returns a date as *time.Time for its value
```